src/item_factory.cpp

```diff
@@ -2667,6 +2681,29 @@ void islot_milling::deserialize( const JsonObject &jo )
     load( jo );
 }
 
+void temperature_effect::load( const JsonObject &jo )
+{
+    optional( jo, was_loaded, "temperature_above", temperature_above );
+    optional( jo, was_loaded, "temperature_below", temperature_below );
+    optional( jo, was_loaded, "transform", transform_into );
+    optional( jo, was_loaded, "remove_poison", remove_poison );
+}
+
+static void load_temperature_effects(
+    const JsonObject &jo, const std::string_view member,
+    cata::value_ptr<islot_temperature_effects> &slotptr, const std::string &src )
```

Comment:
Clang error here. Looks like you pass along `src` but forgot to check if two definitions come from the same source?